AutomatedTesting/Platform/Android/android_project.cmake

```diff
@@ -0,0 +1,8 @@
+# {BEGIN_LICENSE}
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+# {END_LICENSE}
```

Comment:
These `{BEGIN_LICENSE}` and `{END_LICENSE}` blocks should be removed from all these files